k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml

```diff
@@ -144,13 +144,15 @@
     "sha256:f3b6b39a6062328c095337b4cadcefd1612348fdd5190b1dcbcb9b9e90bd8068": ["v1.0"]
     "sha256:cdc56f415500de9a6342be4f8703850dab9eae8870e9f33d9b8598232abd6ac0": ["v1.1"]
     "sha256:64d8c73dca984af206adf9d6d7e46aa550362b1d7a01f3a0a91b20cc67868660": ["v1.1.1"]
+    "sha256:2188def723c8fcb2635fbe54607ac3b4efba581c4fcc3d2279ec9c57da1b2f6f": ["v20220511-helm-chart-4.1.1-1-g24b8e32b0"]
```

Comment:
@longwuyuan please stick with the previous version tag ("v1.2.0")

Reply:
sorry, first time on the certain and errors convention so made wrong assumption. Fixed and pushed now. Please check